### ISSUE TITLE
Show account details

### ISSUE THREAD

**Issue by ilblackdragon** [nearprotocol/near-explorer#53]
If there is @account_id we should make it clickable and lead to page that shows details about this account.

E.g, this type of information:
```
http post https://rpc.nearprotocol.com/ jsonrpc=2.0 method=query params:='["account/register.near", ""]' id="dontcare"
HTTP/1.1 200 OK
Alt-Svc: clear
Via: 1.1 google
content-length: 251
content-type: application/json
date: Wed, 31 Jul 2019 00:11:32 GMT

{
    "id": "dontcare",
    "jsonrpc": "2.0",
    "result": {
        "account_id": "register.near",
        "amount": "99999995350000000000",
        "code_hash": "GKot5hBsd81kMupNCXHaqbhv3huEbxAFMLnpcX2hniwn",
        "nonce": 582,
        "public_keys": [
            "EsjyvmBb2ESGiyjPHMBUnTGCe1P6hPjmxxY2b2hrTBAv"
        ],
        "stake": "0"
    }
}
```

**Comment by frol** [comment]
@robin-thomas I would like to clarify that it seems that @ilblackdragon suggest we query Nearcore RPC directly from the frontend. Instead of manual construction of the RPC calls to Nearcore, I suggest we use `nearlib`; we already use it on the backend:

https://github.com/nearprotocol/near-explorer/blob/03d7540759e198fde70026808a4725f42c17a800/backend/src/index.js#L8-L9

To send a query request, use `nearRpc.query("account/register.near", "")`. Just for the reference, here is the `query` implementation in the `nearlib`:

https://github.com/nearprotocol/nearlib/blob/7c6612b4b677a36ba1fbbdab6a9b20c23245d9ae/src.ts/providers/json-rpc-provider.ts#L42-L48

We don't have the Nearcore RPC URL on the frontend side of things yet, so add it to the `next.config.js` -> `publicRuntimeConfig` with the default `https://rpc.nearprotocol.com` value.

**Comment by robin-thomas** [comment]
@frol Just curious. Why do we need to query nearcore directly from frontend side? I mean, we do have a backend and a database.

**Comment by frol** [comment]
@robin-thomas On the second thought, it seems to be better to implement RPC proxy through WAMP -> backend -> nearcore just for the sake of minimizing the number of moving parts.

The reason I suggested to do the direct calls to frontend is that there is currently no need to do any caching of that information and thus, it is just easier to query nearcore directly for the latest information.

**Comment by frol** [comment]
@robin-thomas I have implemented the proxy in #70.

**Comment by frol** [comment]
Superseded by #90
